### DISCUSSION TITLE
Prometheus two replica pod(k8s) with duplication metrics data.

### DISCUSSION THREAD

**Discussion by babujii** [prometheus/prometheus#9372] [Q&A]
Prometheus two replica pod with duplication metrics data.
          Currently we have two prometheus pod on K8S cluster and collecting metrics to VM servers.
it is working fine. We are thing now two pod going and collecting metrics to VM or only one prometheus pod collecting metrics . We want know.  If two prometheus pod(replica 2) run on the cluster. Will it happen data duplication metrics? can any one help please.

**Comment by roidelapluie** [comment]
Can you please clarify what VM is ? thanks.
